code/modules/mob/living/carbon/species.dm

```diff
@@ -410,6 +410,38 @@
 
 	H.stop_tail_wagging(1)
 
+/datum/species/vulpkanin
+	name = "Vulpkanin"
+	icobase = 'icons/mob/human_races/r_vulpkanin.dmi'
+	deform = 'icons/mob/human_races/r_vulpkanin.dmi'
+	path = /mob/living/carbon/human/vulpkanin
+	default_language = "Galactic Common"
+	language = "Canilunzt"
+	primitive = /mob/living/carbon/monkey/vulpkanin
+	tail = "vulptail"
+	unarmed_type = /datum/unarmed_attack/claws
+	darksight = 8
+	slowdown = -1
+	brute_mod = 1.1
+	burn_mod = 1.1
+
+	blurb = "Vulpkanin are a species of extremely quick canine-pideds residing on the planet Altam just barely within the \
```

Comment:
these blurbs should really be moved to serverside defines of some kind so that lore changes can be made by a server hoster easily and not require code changes